rbac/core/kafka.py

```diff
@@ -35,15 +40,29 @@ class RBACProducer:
     def get_producer(self):
         """Init method to return fake kafka when flag is set to false."""
         if not hasattr(self, "producer"):
+            retries = 0
+            max_retries = 5
```

Comment:
What do you think about making the `max_retries` parameter configurable? That way if the retries need to be adjusted it just would be a matter of updating the Clowder configuration.